pr-1.js

```diff
@@ -0,0 +1,25 @@
+window.currentUser = { id: '19', name: 'Jane', email: 'jane@chameleon.io' };
+
+// users are like authors and profiles like commentors
+// open a modal with commentor info when clicked
+
+
+export const  ActiveProfiles({ profiles, onLaunchProfile }) => {
```

Comment:
It's missing the equal signal (`=`)

```suggestion
export const ActiveProfiles = ({ profiles, onLaunchProfile }) => {
```